### ISSUE TITLE
USB tethering (Android 13)

### ISSUE THREAD

**Issue by ggroloff** [Mygod/VPNHotspot#413]
Android 13 issues:

Enabling USB tethering does not show any clients in the client list in the app.

Standard USB tethering toggle sets the other USB tethering (NCM) toggle to on. 

Setting USB tethering (NCM) toggle to on from off state returns an error:
NCM: TETHER_ERROR_SERVICE_UNAVAIL

Enabling ncm0 toggle while connected to VPN breaks the USB tethering entirely.

Device: Pixel 6a.

**Comment by Mygod** [comment]
Cannot reproduce. Try starting USB tethering from system settings?

**Comment by ggroloff** [comment]
Yes, I have tried that as well. I believe the issue is related to the pixel 6 phones using USB tethering with the NCM protocol, rather than RNDIS. I don't know if all Android 13 phones do this. It's been hard for me to find anything other than anecdotal information regarding it.. but for example: https://support.google.com/pixelphone/thread/161548533/usb-tethering-won-t-work-on-google-6-pro?hl=en

This person has an issue with NCM protocol. I can get USB tethering to work with the NCM drivers however, I just can't get VPN hotspot to work with it on my pixel 6a. Works great on my other devices that use RNDIS.

Thank you for this wonderful app by the way.

**Comment by Mygod** [comment]
Okay I can see the issue now.
